_includes/scripts.html

```diff
@@ -11,19 +11,7 @@
 
 {% if site.ga.ua %}
```

Comment:
This bit of code appears to be gated behind a config setting. So we should use `site.ga.ua` directly. (I'll make this change to the PR)